Tasks/Birich/Task4/Task4.py

```diff
@@ -0,0 +1,58 @@
+while True:
+    answer = input(
+        "\nWould you like to convert the text to the maximum number of characters per line starting from 36 characters:\n\
+    Yes, I would like to - 'yes/Y'\n    No, I don't want to - 'no/N'\n    enter an option: ")
+
+    variable_yes, variable_Y = "yes", "Y"
+    variable_no, variable_N = "no", "N"
+
+    if answer == variable_no or answer == variable_N:
+        print("\nWell, it's your choice! If anything, call the function yourself...")
+    elif answer != variable_yes and answer != variable_Y:
+        print(
+            f"\nYou have introduced some kind of heresy - {answer}, read the condition carefully and make your choice!\n")
+        continue
+    else:
+        # I don't use a chunk since the file is known to me
+        with open("text.txt", 'r') as text_original:
+            with open("text2.txt", 'w') as text_modified:
+                data_text = text_original.read()
```

Comment:
btw since you're reading the whole file there is no reason for you to hold it open while working with the second one